### ISSUE TITLE
Incorrect path? ... build/graalvm/graalvm-ce-java11-20.2.0/Contents/Home/bin/gu

### ISSUE THREAD

**Issue by rbygrave** [formkiq/graalvm-native-plugin#4]
```* What went wrong:
Execution failed for task ':graalvmNativeImage'.
> A problem occurred starting process 'command '/home/rob/github/other/example-graal/hello/build/graalvm/graalvm-ce-java11-20.2.0/Contents/Home/bin/gu''
```

So this path does not exist.  What does exist is `build/graalvm/graalvm-ce-java11-20.2.0/bin/gu`   
So removing the `Contents/Home` part makes the path correct.

**Comment by mfriesen** [comment]
Can you please provide your nativeImage { ... } block and the platform you are using, e.g mac/linux/windows.

**Comment by rbygrave** [comment]
Using linux, my nativeImage block was:
```
nativeImage {
    mainClassName = 'com.example.Application'
    enableHttp          = false
    enableHttps         = false
    reflectionConfig    = "$buildDir/classes/java/main/META-INF/graal/reflect.json"
}
```

I played around some more to get to:
```
nativeImage {
    mainClassName = 'com.example.Application'
    imageFile = '/home/rob/github/other/example-graal/hello/build/graalvm/graalvm-ce-java11-20.2.0/bin/gu'
    enableNoFallback = true
    enableDocker = true                     // was hitting an error with tag unzip without this ...
    enableHttp          = false
    enableHttps         = false
    reflectionConfig    = "$buildDir/classes/java/main/META-INF/graal/reflect.json"
}
```
Sorry, I have got busy again.  I'll hopefully play more again later and get you more details on the tar.gz uncompress error

Cheers, Rob.

**Comment by mfriesen** [comment]
I noticed in my sample app was using version 1.0.0 of the plugin. The 2nd problem you ran into with the requiring "enableDocker = true" is fixed if you use 1.0.1.

**Comment by mfriesen** [comment]
Issue fixed in version 1.0.2
